lib/matplotlib/artist.py

```diff
@@ -898,6 +900,89 @@ def set_zorder(self, level):
         self.pchanged()
         self.stale = True
 
+    def get_top_margin(self):
```

Comment:
So there's only one property we currently whitelist: `axes`.  I guess I'm willing to put this up to a "vote", but I'm strongly in favor of staying with setters/getters for now until we move over to traitlets *en masse*.  There's just too much source for confusion otherwise.